web/sites/default/config/language/es/views.view.project_map.yml

```diff
@@ -28,28 +28,6 @@ display:
             suffix: null
             more_link_text: null
           separator: ','
-        field_geo_map:
```

Comment:
Was the removal of these two fields from this view (and other translated views) intentional? On the English view, the field name changed from `field_geo_map` to `field_geo_map_string`, but here the fields are removed instead of being renamed.

Reply:
Sorry for the confusion. This view appears to be an experimental view that never been linked to. In this PR, it was disabled but not removed. I might have updated the view before disabling it which is the source of the confusion.

Reply:
On a second thought, it may be cleaner to delete the view in this case. I made another push.